.github/workflows/tests.yml

```diff
@@ -50,12 +50,12 @@ jobs:
           - name-suffix: "(Minimum Versions)"
             os: ubuntu-22.04
             python-version: '3.11'
-            extra-requirements: '-c requirements/testing/minver.txt'
+            extra-requirements: 'python -m pip install --upgrade $PRE --group test-minver'
```

Comment:
Why the addition of `$PRE` here?

Reply:
This was always used: https://github.com/matplotlib/matplotlib/pull/29281/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fL218

But I've changed this line a bit to not require the full invocation.